pkg/transport/listen_windows.go

```diff
@@ -25,6 +25,8 @@ func Listen(endpoint string) (net.Listener, error) {
 			VMID:      hvsock.GUIDWildcard,
 			ServiceID: svcid,
 		})
+	case "unix", "unixpacket":
```

Comment:
I'm not sure `unixpacket` is desirable here, this was added for BESS support: https://github.com/containers/gvisor-tap-vsock/commit/826b1b6f6fb3017cf8c47891b7cfad3101a561ee
macOS only supports "unix" and not "unixpacket"

Reply:
I don't know the details of "unixpacket" thing. Though of it as some kind of alias. Lack of knowledge here. Changed it to "unix" only.